parent/pom.xml

```diff
@@ -389,6 +389,10 @@
     <jettison-version>1.4.0</jettison-version>
     <jgit-version>5.1.3.201810200350-r</jgit-version>
     <jgroups-version>4.0.15.Final</jgroups-version>
+    <jgroups-raft-version>0.4.2.Final</jgroups-raft-version>
+    <jgroups-raft-jgroups-version>4.0.15.Final</jgroups-raft-jgroups-version>
```

Comment:
We can use directly the jgroups-version in this case

Reply:
The problem is that in the future the 2 might differ, jgroups-raft is not tided to jgroups releases so...

Reply:
Ok